tests/test_save_bam_mapped.yml

```diff
@@ -5,26 +5,23 @@
     - save_bam_mapped
   files:
     - path: results/multiqc
-    - path: results/preprocessing/1234N/mapped/1234N.bam
-    - path: results/preprocessing/1234N/mapped/1234N.bam.bai
-    - path: results/preprocessing/1234N/markduplicates/1234N.md.cram
-    - path: results/preprocessing/1234N/markduplicates/1234N.md.cram.crai
-    - path: results/preprocessing/1234N/recal_table/1234N.recal.table
-    - path: results/preprocessing/1234N/recalibrated/1234N.recal.cram
-    - path: results/preprocessing/1234N/recalibrated/1234N.recal.cram.crai
+    # - path: results/preprocessing/test/mapped/test-test_L1.bam
```

Comment:
Need to keep the bam file for samples with a single pair of fastq

Reply:
Do we need to create the single end cram file? I can do that if you need it

Reply:
No, no issue there